cmd/update/project.go

```diff
@@ -65,23 +67,18 @@ Usage
 `
 )
 
-var (
-	projectConfig      = &ProjectConfig{}
-	errProjectNotFound = "Project %v not found\n"
-	errInvalidUpdate   = "Invalid state passed. Specify either activate or archive\n"
-	errFailedUpdate    = "Project %v failed to get updated to %v state due to %v\n"
-)
+var projectConfig = &ProjectConfig{}
 
 func updateProjectsFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
 	id := config.GetConfig().Project
 	if id == "" {
-		fmt.Printf(errProjectNotFound, id)
+		fmt.Printf(clierrors.ErrProjectNotPassed)
```

Comment:
why not return an error with the same message?

Reply:
Missed it . Yes we should be returning this error.
Fixed it.